data/human/culture conversations.txt

```diff
@@ -211,6 +241,22 @@ mission "Local Politics"
 
 
 
+mission "Military Response Test"
+	minor
+	source
+		government "Free Worlds"
+		not attributes "station" "urban" "frontier"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`In a strange set of directions, the Air Traffic Controller had you punch in a set of coordinates taking you close to a massive military base on the way to the landing zone. You let the autopilot take over as you watch the view off the cockpit window, hoping the turrets in the base wouldn't have you shot out of the sky.`
```

Comment:
Not sure about "a strange set of directions".
```suggestion
			`In a strange turn of events, the Air Traffic Controller has you punch in a set of coordinates, taking you close to a massive military base on the way to the landing zone. You let the autopilot take over as you watch the view off the cockpit window, hoping the turrets in the base won't shoot you out of the sky.`
```

Reply:
"Strange set of instructions," perhaps. I don't want to use "Strange turn of events" because it didn't exactly feel right. 
It's also not illegal to fly over a military base. One close enough to a spaceport.

Reply:
```suggestion
			`In a strange set of instructions, the Air Traffic Controller has you punch in a set of coordinates, taking you close to a massive military base on the way to the landing zone. You let the autopilot take over as you watch the view off the cockpit window, hoping the turrets in the base won't shoot you out of the sky.`
```